server/player/player.go

```diff
@@ -1939,6 +1941,20 @@ func (p *Player) Breathing() bool {
 	return !submerged
 }
 
+//Cps will return the current Cps of the player
+func (p *Player) Cps() int {
+	return p.cps
+}
+
+//addCps updates the Cps of the player and removes it 1 second later
+func (p *Player) addCps() {
+	go func() {
+		p.cps += 1
+		time.Sleep(1 * time.Second)
+		p.cps -= 1
+	}()
+}
+
```

Comment:
Please use proper punctuation and formatting. Running ``go fmt`` over your code should fix the formatting issues. Acronyms should be all capitals.